### ISSUE TITLE
How to change the style without selecting first

### ISSUE THREAD

**Issue by bobo19870225** [pChochura/richtext-compose#1]
How to change the style without selecting first

**Comment by joshuaorotayo** [comment]
Hi,
Thank you for the library!

I want to save the Rich Text value and its formatting in a room database. Is there a way to do this, please?

**Comment by joshuaorotayo** [comment]
> How to change the style without selecting first

Did you ever manage to do this?

**Comment by pChochura** [comment]
@bobo19870225 Sadly it is not possible. The default behavior is to change the style of the current composed word (or sentence) highlighted be the underline (it's IME feature).

@joshuaorotayo Yes, there's a way. I need to update the docs. You can use [RichTextValueSnapshot](https://github.com/pChochura/richtext-compose/blob/main/library/src/main/java/com/pointlessapps/rt_editor/utils/RichTextValueSnapshot.kt) which is fully serializable.
```kotlin
// Create a new content value
val content: RichTextValue = RichTextValue.get()
// Change it with the editor as you wish and get the last snapshot
val snapshot: RichTextValueSnapshot = content.getLastSnapshot()
// You can save the snapshot to the database

// Get the content value back. It will render the content the same way as with the [content] value
val contentFromSnapshot = RichTextValue.fromSnapshot(snapshot)
```

**Comment by joshuaorotayo** [comment]
Thanks for the response, sorry im still confused.

I have a note object and want to save the richContent. This is what i currently have and then I've made a typeconverter to retrieve the value but it doesn't seem to be working.

How exactly would i get it to use the type converter

![image](https://user-images.githubusercontent.com/9823193/176668709-363084d0-5183-4de4-841c-59464fc44282.png)

![image](https://user-images.githubusercontent.com/9823193/176668881-2726841b-86e6-4a87-8f05-f5238b55f9f5.png)

**UPDATE**

Found out how to do it. My Room database wasn't updating correctly and just needed to put the type converter annotation on just my database code then Uninstall the app from emulator and "Make" the project again

![image](https://user-images.githubusercontent.com/9823193/176866406-c1ae6c21-55b7-4d98-8148-0e0dba0ecfb2.png)

![image](https://user-images.githubusercontent.com/9823193/176866903-d7708702-5a80-4ecf-be79-e3aaab27726e.png)
